docs/openapi/appio/schemas-pn-timeline-appio-v1.yaml

```diff
@@ -735,36 +735,44 @@ components:
             * `PG`
         taxId:
           $ref: '../schemas-pn-notification-v1.yaml#/components/schemas/TaxId'
-        internalId:
-          type: string
-          description: id interno anonimizzato
         denomination:
           $ref: '../schemas-pn-notification-v1.yaml#/components/schemas/Denomination'
-        digitalDomicile:
-          $ref: "#/components/schemas/NotificationDigitalAddress"
-        physicalAddress:
-          $ref: "../schemas-pn-notification-v1.yaml#/components/schemas/NotificationPhysicalAddress"
         payment:
-          $ref: "../schemas-pn-notification-v1.yaml#/components/schemas/NotificationPaymentInfo"
+          $ref: '#/components/schemas/NotificationPaymentInfo'
```

Comment:
il recipientType ho pensato di non eliminarlo dallo schema in quanto required. Se non è giusto, elimino anche quello.

Reply:
ok, visto che è string e non Enum, va bene.